UnitedIncome/components
# [APP-4910] Slider Tooltip Style Updates

### Description
- Updated the slider tooltip height to be flexible based on the amount of content (but with a max height of 10rem still, which I cleared with Monique).
- Fixed positioning of the first and last positions for smaller viewports

### Testing Instructions
Check that the slider tooltip updated correctly based on the specs but that the non-slider tooltips have no visual change.

### Screenshots
Full width & one-liner tooltip:
[![Image from Gyazo](https://i.gyazo.com/bac8b80d2e984f40394e77a4b9c327ef.gif)](https://gyazo.com/bac8b80d2e984f40394e77a4b9c327ef)

----------------------
Tablet viewport & bigger tooltip:
[![Image from Gyazo](https://i.gyazo.com/2cd0baf535897fcbcdd916120d945f38.gif)](https://gyazo.com/2cd0baf535897fcbcdd916120d945f38)

-----------------------
Small viewport:
![image](https://user-images.githubusercontent.com/8313062/81438506-e38caf00-913a-11ea-9c67-b3df5322a803.png)

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4910)
